### ISSUE TITLE
[Bug]: vmix emulation not working

### ISSUE THREAD

**Issue by matiaspl** [josephdadams/TallyArbiter#420]
### What happened?

No sign of vmix emulation listener port opened. After downgrading to v2 stable, clearing and recreating the config vmix emulation works fine (and of course the 8099 port is visible in netstat's output). 

### Version

3.0.6

### Distribution

Desktop

### OS

Windows

**Comment by JTF4** [comment]
Just to confirm, vmix is working after resetting the config?

If that's the case, it's because the V3 config file is not 100% backwards comparable with V2. It has more sections which can cause some issues when starting V2

**Comment by josephdadams** [comment]
No I think he’s saying v2 it works, but not in v3 at all

**Comment by matiaspl** [comment]
That's correct. Clean install v3 - vmix clients don't work. Clean install v2 - vmix clients work fine.

**Comment by josephdadams** [comment]
The overall feature stack is not drastically different in the two versions, so I would just recommend sticking with v2 for now until we can get that sorted out.

**Comment by Smolinc** [comment]
I also confirm it does not work on version 3.0.3. I returned to version 2.0.16, everything works stably.
